installer/functions/account/get-account-info.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+
+export const handler: ServerlessFunctionSignature = async function(
+  context: Context,
+  event: {},
+  callback: ServerlessCallback
+) {
+  const client = context.getTwilioClient();
+  const response = new Twilio.Response();
+  response.appendHeader('Access-Control-Allow-Origin', '*');
+  try {
+    const accounts = await client.api.accounts
+      .list({status: 'active', limit: 20})
+      .then(accounts => accounts);
+  
+    if (accounts.length) {
```

Comment:
consider writing code with guard statements, it reduces cognitive load as you can exit early with error cases, leaving the happy as the last statement of your function. in this example, if you inverted this to 

```tsx
    if (!accounts.length) {
      throw new Error("No Accounts found!");
    }
    // rest of your code
```

you could remove one layer of nesting and leave errors for the end

https://www.catalyst.net.nz/blog/early-returnsguard-clauses-javascript-and-react